### PR TITLE
To drop root privilege

### PR DESCRIPTION
**Pull request by krish-1**
Privilege.c file is added to drop privilege for parodus process

### CODE REVIEW

**Comment by krish-1** [comment]
To drop root privilege to parodus process

**Comment by krish-1** [comment]
Pull request created

File: src/privilege.h [hunk 1]
```diff
@@ -0,0 +1 @@
+void drop_root_privilege();
```
Comment (shilpa24balaji) [inline src/privilege.h:1]: Please add some comments on the use of this and TODO

**Comment by shilpa24balaji** [comment]
@krish-1 17 commits for 4 file changes is not good. Could you please use new branch use 1 commit and then submit PR.
Please add basic comments in file for the use of this feature. 
Also please modify the CHANGELOG.md with one line comment on the change.

**Pull request closed without merging**